parlai/agents/drqa/drqa.py

```diff
@@ -190,7 +190,7 @@ def act(self):
         if ex is None:
             return reply
         batch = batchify(
-            [ex], null=self.word_dict['<NULL>'], cuda=self.opt['cuda']
+            [ex], null=self.word_dict['__NULL__'], cuda=self.opt['cuda']
```

Comment:
Instead of hard coding it again, let's just use self.word_dict[self.word_dict.null_token]